pom.xml

```diff
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <url>https://wiki.jenkins-ci.org/display/JENKINS/Spotinst+Plugin</url>
+    <url>https://github.com/jenkinsci/spotinst-plugin/JenkinsWiki.adoc</url>
```

Comment:
This is a non-existent URL. It will not work.
You wanted to use https://github.com/jenkinsci/spotinst-plugin/blob/master/JenkinsWiki.adoc , I'd guess